### DISCUSSION TITLE
`pyenv --version` returns previous version number

### DISCUSSION THREAD

**Discussion by cesarcoatl** [pyenv/pyenv#2227] [General]
I have updated to the latest version (`2.2.4`) via `brew`, but running `pyenv --version` prints `2.2.3`.

Code:
https://github.com/pyenv/pyenv/blob/8aa8d84e0961fbea6dea28c175db0a5da7bf49c5/libexec/pyenv---version#L15

Expected result:

```bash
$ pyenv --version
pyenv 2.2.4
```

Actual result:

```bash
$ pyenv --version
pyenv 2.2.3
```

Please let me know if this should be submitted as an issue.

CC: @native-api

**Comment by cesarcoatl** [comment]
Formula: [`pyenv.rb`](https://github.com/Homebrew/homebrew-core/blob/4ebc58b38d2960f746579e2072b05ce46bedfef4/Formula/pyenv.rb)

```bash
$ brew info pyenv
pyenv: stable 2.2.4 (bottled), HEAD
Python version management
https://github.com/pyenv/pyenv
/usr/local/Cellar/pyenv/2.2.4 (859 files, 2.9MB) *
  Poured from bottle on 2022-01-26 at 15:06:49
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/pyenv.rb
License: MIT
==> Dependencies
Required: autoconf ✔, openssl@1.1 ✔, pkg-config ✔, readline ✔
==> Options
--HEAD
	Install HEAD version
==> Analytics
install: 100,312 (30 days), 369,690 (90 days), 1,546,057 (365 days)
install-on-request: 98,920 (30 days), 364,350 (90 days), 1,516,745 (365 days)
build-error: 1 (30 days)
```

**Reply by laz-** [reply]
looks like the version was just updated in [6f7a29c3f4a4a5931a28666ec02c262546f333a0](https://github.com/pyenv/pyenv/commit/6f7a29c3f4a4a5931a28666ec02c262546f333a0#diff-02337a5abda801c8413a983af6c0a42877407537f41ee1bcce34c302deebc807)
